web/src/test/java/com/epam/rft/atsy/cucumber/welcome/WelcomeStepDefs.java

```diff
@@ -24,9 +24,12 @@
 
   @When("the user clicks on the Főoldal button")
```

Comment:
"Főoldal button" does not exist anymore, so update this When annotation's value string. Be careful, because you have to modify this on the feature file too.